### PR TITLE
add julia merly example

### PR DESCRIPTION
**Pull request by codeneomatrix**

### CODE REVIEW

File: Julia-merly/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM julia:1.0.0-stretch as base
```
Comment (TooTallNate) [inline Julia-merly/Dockerfile]: I think you don't want `as base` here

File: Julia-merly/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+WORKDIR /usr/src
+COPY . .
+RUN julia -e "using Pkg; Pkg.add(\"Merly\")"
+Expose 8080
```
Comment (TooTallNate) [inline Julia-merly/Dockerfile]: `EXPOSE` actually isn't necessary for Cloud v2 deployments.

File: Julia-merly/Dockerfile (hunk 1 above)
Comment (TooTallNate) [reply Julia-merly/Dockerfile]: > ▲ The built image size (188.4M) exceeds the 100MiB limit

Looks like this base image is too big. I suggest finding an Alpine-based image. Perhaps [cmplopes/alpine-julia](https://hub.docker.com/r/cmplopes/alpine-julia/).

**Comment by codeneomatrix** [comment]
I'm sorry, at this time I can not make the changes; please close this issuse. Thank you

**Comment by rauchg** [comment]
Thanks @codeneomatrix – we'll try to take it up from here

**Comment by joecohens** [comment]
I'll fix it, thank you @codeneomatrix!

**Comment by joecohens** [comment]
Thank you for taking the time to make this PR! We are closing it for now since it is not applicable to our newest release Now 2.0, you can read more about the new release https://zeit.co/blog/now-2

**Pull request closed without merging**